pkgs/leak_tracker/lib/src/leak_tracking/_primitives/_print_bytes.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:intl/intl.dart';
+
+String? prettyPrintBytes(
```

Comment:
Is this code duplicated from `lib/src/shared/primitives/utils.dart`?

Reply:
Yes, it is copied from DevTools.
Maybe in future this code should be published. Maybe leak_tracker is a right package to publish it.

But for now I do not want to make it part of API and add toil of watching backward compatibility.

Reply:
Right, I totally blanked and thought this was in the DevTools repo.